src/contrib/mmap_cache.rst

```diff
@@ -439,6 +442,38 @@ number of GIDs ``num_groups`` ``0x4`` (4). The GIDs of the groups are
 ``0x2bb9d92a`` (733600042), ``0x2bb9d933`` (733600051), ``0x2bb9d934``
 (733600052) and ``0x2bb9d935`` (733600053).
 
+The SID Data
+^^^^^^^^^^^^
+
+The SID data is defined in ``src/util/mmap_cache.h`` as:
+
+.. code-block:: c
+
+    struct sss_mc_sid_data {
+        rel_ptr_t name;         /* ptr to SID string, rel. to struct base addr */
+        uint32_t type;          /* enum sss_id_type */
+        uint32_t id;            /* gid or uid */
+        uint32_t populated_by;  /* 0 - by_id(), 1 - by_uid/gid() lookup */
+        uint32_t sid_len;       /* length of sid */
+        char sid[0];
+    };
+
+The ``name`` pointer is a shortcut to the ``sid`` data and is used to make sure
+that the object so far only found with the help of the hash value does match the
+input SID. If the user is search by ID the ``id`` value is used for this check.
```

Comment:
The part which begins with `and is used...` and ends with `match the input SID` is a bit confusing to me. Below text sounds a bit better to me, but please validate it.

The ``name`` pointer is a shortcut to the ``sid`` data and is used to make sure that the object, found with the hash value, matches the input SID.

Reply:
Yes, this is better, thank you. (I just took a template from description of other types).